proto/buf/alpha/registry/v1alpha1/admin.proto

```diff
@@ -35,6 +35,45 @@ message BreakingChangePolicy {
   BreakingChangeCategory category = 3;
 }
 
+// UniquenessPolicy is the policy for uniqueness on the server.
+message UniquenessPolicy {
+  // Whether uniqueness policy is enforced.
+  bool enabled = 1;
+}
+
+// Collision is a collision that has occurred.
+message Collision {
+  // The reference that has collisions.
+  string reference = 1;
+  // The type of collision.
+  CollisionType type = 2;
+  // The sources of the collision that the user is authorized to see.
+  repeated CollisionSource sources = 3;
+  // The number of collisions that were not returned.
+  // The user should contact a server admin for more information about these collisions.
+  uint64 num_other_collisions = 4;
```

Comment:
👍 on thinking about this, but maybe we should just start with `has_other_collisions` to keep things as locked down as possible to start with?

It also feels a little weird to me that a `message Collision` has a field named `num_other_collisions`, maybe it should be `has_other_collision_sources`?